src/assets/icons/icons.svg

```diff
@@ -32,6 +32,13 @@
       <desc>Copy Icon</desc>
       <path d="M7.723 2.099H2.144a.95.95 0 00-.948.948v8.004a.95.95 0 00.948.949h5.579a.95.95 0 00.949-.949V3.047a.952.952 0 00-.95-.948zm.283 8.95a.285.285 0 01-.286.285H2.142a.285.285 0 01-.285-.285V3.047c0-.157.128-.285.285-.285H7.72c.158 0 .286.128.286.285v8.002z M9.856 0H4.278a.95.95 0 00-.949.949.33.33 0 00.332.331.33.33 0 00.332-.331c0-.158.127-.285.285-.285h5.578c.158 0 .285.127.285.285v8.004a.285.285 0 01-.285.285.33.33 0 00-.332.332.33.33 0 00.332.331.95.95 0 00.949-.948V.949A.95.95 0 009.856 0z"/>
     </symbol>
+    <!-- REFACTOR EDIT ICON -->
```

Comment:
@PinhoOn I think this is not updated from master branch cause @felipprodrigues removed this commented icon.